cmd/snap/main.go

```diff
@@ -31,6 +31,7 @@ import (
 	"unicode/utf8"
 
 	"github.com/jessevdk/go-flags"
+	"github.com/makholm/covertool/pkg/exit"
```

Comment:
Please make sure that the new 3rd dependencies we introduce are available in distributions that require us to use packages from their repositories and disallow vendoring. Those are Fedora and Debian. Otherwise it means we'd first have to get those packages into the respective distros and backport them to supported releases. Meaning, it will take a **very** long time.